src/lib/blocks.tsx

```diff
@@ -52,6 +57,13 @@ export const blockifyHTML = (htmlContent: string): JSX.Element[] => {
 }
 
 const blockifyElement = (element: HTMLElement): JSX.Element => {
+  if (element.classList.contains(OS_RAISE_IB_CTA_CLASS)) {
+    const maybeCTABlock = parseCTABlock(element)
```

Comment:
Minor: If you wanted to get coverage on this  you can extend the `blockifyHTML handles non-block content with interactive blocks` test case similar to what I did in [this PR](https://github.com/openstax/k12-apps-raise/pull/10). (arguably perhaps I should have created a separate test case instead of hijacking that one :man_shrugging: )